lib/ssl/test/ssl_basic_SUITE.erl

```diff
@@ -782,6 +786,28 @@ incompleat_chain_length(Config) when is_list(Config)->
                                         {options, [{verify, verify_peer}, {verify_fun, VerifyFun} | ClientConf]}]),
     ssl_test_lib:check_result(Client, ok, Server, ok).
 
+
+user_dies() ->
+    [{doc, "Test that we do no leak processess when user dies during startup of connection"}].
+user_dies(Config) when is_list(Config) ->
+    process_flag(trap_exit, true),
+    ServerOpts = ssl_test_lib:ssl_options(server_rsa_verify_opts, Config),
+    ClientOpts = ssl_test_lib:ssl_options(client_rsa_verify_opts, Config),
+    {_, ServerNode, Hostname} = ssl_test_lib:run_where(Config),
+
+    Port = ssl_test_lib:inet_port(ServerNode),
+    Server = spawn_link(fun() ->
+                                {ok, L} = ssl:listen(Port, ServerOpts),
+                                loop(L)
+                        end),
+    {ok, _} = ssl:connect(Hostname, Port, ClientOpts),
+    2 = proplists:get_value(supervisors, supervisor:count_children(tls_connection_sup)), 
```

Comment:
It could have been replaced by check_process_count/0, but I added an argument to the function to keep the knowledge of the expected count in the main test case function.

Reply:
yes. I meant you could replace line 804 with check_process_count(0) call and reduce code repetition ...